spec/features/listing_all_peeps_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'spec_helper'
+require_relative 'helpers/session'
+
+include SessionHelpers
```

Comment:
You're including SessionHelpers in a global scope here... which could be disastrous in a large codebase. Within your RSpec configuration, have "$ config.include(SessionHelpers)" instead.